ui/lib/pki/addon/components/pki-generate-csr.ts

```diff
@@ -70,6 +71,9 @@ export default class PkiGenerateCsrComponent extends Component<Args> {
         const useIssuer = yield this.getCapability();
         yield model.save({ adapterOptions: { actionType: 'generate-csr', useIssuer } });
         this.flashMessages.success('Successfully generated CSR.');
+        if (this.args.onSave) {
```

Comment:
if this arg doesn't exist do they just stay on the current view? Do we need to handle that situation at all?

Reply:
The intention here is that they should stay on the current view, but `onSave` can be called for side effects. typically in these cases it's to update the page title on the parent when the save is successful. Maybe updating the naming could make that more clear?

Reply:
I think this get's answered in the later documentation of pki-generate-root.ts. It's what made me think of the documentation comment.